hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +43,24 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB mass storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB mass storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
```

Comment:
Just out of curiosity, is this because the "USB host boot mode" OTP bit is already set on the Pi 3B+, or is it because the bootcode in the BCM2837 doesn't check for this bit?
If it's the former, perhaps "The Raspberry Pi 3B+ and CM3+ support USB mass storage boot out of the box, because the host boot mode OTP bit is already set." would clarify this? (or perhaps that's unnecessary extra explanation?) :man_shrugging: